From 1db4d07bb78858da7676720d13ee7c4f3bd8e6e0 Mon Sep 17 00:00:00 2001
From: Nate Graham <nate@kde.org>
Date: Sun, 12 Jan 2020 08:39:29 -0700
Subject: Revert "[KCMs/Activities] Fix jagginess for activities list icons on
 high DPI systems"

This reverts commit 567f71714faac305947573e307f486a9960cf1d1.

The commit in question was pushed without review and should not have
been. It worked fine for me in local testing but somehow was broken
anyway. Sorry. :(

BUG: 416158
---
 kcms/activities/qml/activitiesTab/ActivitiesView.qml | 2 --
 1 file changed, 2 deletions(-)

diff --git a/kcms/activities/qml/activitiesTab/ActivitiesView.qml b/kcms/activities/qml/activitiesTab/ActivitiesView.qml
index 531ef9f..aac13a2 100644
--- a/kcms/activities/qml/activitiesTab/ActivitiesView.qml
+++ b/kcms/activities/qml/activitiesTab/ActivitiesView.qml
@@ -52,8 +52,6 @@ ColumnLayout {
                         height: Kirigami.Units.iconSizes.medium
                         width: height
                         source: model.icon
-                        sourceSize.width: model.icon.width
-                        sourceSize.height: model.icon.height
                     }
 
                     QQC2.Label {
-- 
cgit v1.1
